src/main/java/net/glowstone/block/itemtype/ItemMilk.java

```diff
@@ -0,0 +1,34 @@
+package net.glowstone.block.itemtype;
+
+import net.glowstone.EventFactory;
+import net.glowstone.entity.GlowPlayer;
+import org.bukkit.Material;
+import org.bukkit.event.player.PlayerItemConsumeEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+
+public class ItemMilk extends ItemFood {
+
+    public ItemMilk() {
+        super(0, 0);
+    }
+
+    @Override
+    public boolean eat(GlowPlayer player, ItemStack item) {
+        PlayerItemConsumeEvent event1 = new PlayerItemConsumeEvent(player, item);
+        EventFactory.callEvent(event1);
+        if (event1.isCancelled()) return false;
+
+        // todo implement playeritemconsumeevent set item
+
+        player.setUsageItem(null);
+        player.setUsageTime(0);
+        player.getInventory().getItemInHand().setType(Material.BUCKET);
+
+        for(PotionEffect potionEffect : player.getActivePotionEffects()) {
```

Comment:
I don't call super, because:
1. The super method removes one item and I would add this item again, this makes no sense.
2. The FoodLevelChange isn't called, when I drink a milk bucket.